README.md

```diff
@@ -0,0 +1,70 @@
+<p align="center">
```

Comment:
Please replace the whole README to be related to `plugin-file-cloud-storage`; ie, Features, Pre-requisite, How to Install, Contribution guideline, etc.

The parts that must be included in this PR are badges showing codecov badge, build badge and renovate badge.

Reply:
Also need to mention this version is WIP, there is no functionality yet. So ppl not trying to use it at the moment.